src/main/java/org/sonarlint/intellij/ui/SonarLintRulePanel.java

```diff
@@ -73,17 +93,35 @@ public void setRuleKey(@Nullable LiveIssue issue) {
       }
 
       StringBuilder builder = new StringBuilder(description.length() + 64);
-
       builder.append("<h2>")
-        .append(issue.getRuleName())
-        .append("</h2>")
-        .append(issue.getRuleKey())
-        .append("<br />")
+        .append(StringEscapeUtils.escapeHtml(issue.getRuleName()))
+        .append("</h2>");
+      createTable(issue, builder);
+      builder.append("<br />")
         .append(description);
       updateEditor(builder.toString());
     }
   }
 
+  private void createTable(LiveIssue issue, StringBuilder builder) {
+    // apparently some css properties are not supported
+    String imgAttributes = "valign=\"top\" hspace=\"3\"";
+
+    builder.append("<table><tr>");
+    if (issue.getType() != null) {
+      builder.append("<td>").append("<img " + imgAttributes + " src=\"file:///type/").append(issue.getType()).append("\"/></td>")
+        .append("<td class=\"pad\"><b>").append(clean(issue.getType())).append("</b></td>");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "</b></td>" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)